tests/basic_correctness/test_chunked_prefill.py

```diff
@@ -8,23 +8,34 @@
 """
 import pytest
 
-from ..models.utils import check_outputs_equal
+from ..models.utils import check_logprobs_close, check_outputs_equal
 
-MODELS = [
+EXACT_MATCH_MODELS = [
     "facebook/opt-125m",
     "meta-llama/Llama-2-7b-hf",
 ]
 
+E5M2_KV_MODELS = [  # type: ignore
+    # does not work with fp8 kv cache kernel
+    # - CUDA illegal memory access - undiagnosed
+    # "facebook/opt-125m",
+]
```

Comment:
Does other models work with E5M2? For example does that work if you just use the FP16 `Qwen2-1.5B-Instruct` checkpoint?

Reply:
I can't seem to find any model that has quantized kv E5M2; if not explicitly quantized with KV scales, the log probs differ due to the inaccuracy during low-precision float conversion

Reply:
E5M2 doesn't need scaling factors and the accuracy shouldn't drop significantly. If it doesn't make sense to check the logprobs for E5M2, we could skip testing them.

Reply:
I see, the reason the test failed was because Qwen2-1.5B is actually BF16.
I tried with llama-2 which is FP16 and the tests pass now/